base_manifest_extension/hooks.py

```diff
@@ -1,21 +1,32 @@
 # -*- coding: utf-8 -*-
 # Copyright 2017 Therp BV <http://therp.nl>
+# Copyright 2017 LasLabs Inc.
 # License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html)
 
 import inspect
 from openerp.sql_db import Cursor
 from openerp.modules import module
 from openerp.modules.graph import Graph
+
 original = module.load_information_from_description_file
+rdepends_to_process = {}
```

Comment:
are multithreaded servers still a thing? then this would be problematic

Reply:
Definitely. Maybe a [`werkzeug.local`](http://werkzeug.pocoo.org/docs/0.12/local/)

Reply:
Ok. I'll go with a ``werkzeug.local``. I try to avoid globals in general, but there didn't seem to be any other option here for retaining info between ``load_information_from_description_file`` calls.